openadmet/models/active_learning/committee.py

```diff
@@ -29,6 +36,177 @@ def from_models(cls, models: list = []):
         )
         return instance
 
```

Comment:
Easy property to check might be nice. 

```suggestion
@property
def calibrated(self):
    return self.calibration_model is None```

Reply:
Done.